torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -3020,6 +3020,40 @@ RegisterOperators reg2({
           return 0;
         },
         aliasAnalysisFromSchema()),
+    Operator(
+        "aten::all(int[] self) -> bool",
+        [](Stack& stack) {
+          c10::List<int64_t> l = pop(stack).toIntList();
+          push(stack, std::move(l));
+          for(int i = 0; i < l.size(); i++){
+            if(l[i] == 0){
+              push(stack, false);
+              break;
+            }
+            else{
+              push(stack, true);
```

Comment:
We don't want to push on to the stack for every iteration. Generally it's easier to reason about the operator if it computes the result and then only does one single stack push at the end of the function.

Reply:
This should only push true onto the stack once, instead of every iteration

Reply:
I am amending this at the moment.

Reply:
Silly error on my part here. I have changed it so true is pushed on the stack before the loop check to see if there is a false value. If there is false is pushed.